src/app-layout/utils/use-drawers.ts

```diff
@@ -19,6 +19,7 @@ interface ToolsProps {
   tools: React.ReactNode | undefined;
   onToolsToggle: (newOpen: boolean) => void;
   ariaLabels: AppLayoutProps.Labels | undefined;
+  disableDrawersMerge?: boolean;
```

Comment:
Current classic and refresh app layouts support tools as special case.

This code allows the toolbar app layout to not support this special case. It will only render `drawers` prop and tools will be auto merged using this code

Decided to go with `disableDriversMerge: false` as default because I am planning to remove this code from other app layouts too and this will be the new default for all cases: https://github.com/cloudscape-design/components/pull/2315 (currently closed, but I will resume that work)